doc/source/cluster/kubernetes/user-guides/configuring-autoscaling.ipynb

```diff
@@ -0,0 +1,1147 @@
+    "\n",
+    "* `detached_actor.py` is a Python script that creates a detached actor which requires 1 CPU.\n",
+    "  ```py\n",
+    "  import ray\n",
+    "  import sys\n",
+    "\n",
+    "  @ray.remote(num_cpus=1)\n",
+    "  class Actor:\n",
+    "    pass\n",
+    "\n",
+    "  ray.init(namespace=\"default_namespace\")\n",
+    "  Actor.options(name=sys.argv[1], lifetime=\"detached\").remote()\n",
+    "  ```\n",
+    "\n",
+    "* `terminate_detached_actor.py` is a Python script that terminates a detached actor.\n",
+    "  ```py\n",
+    "  import ray\n",
+    "  import sys\n",
+    "\n",
+    "  ray.init(namespace=\"default_namespace\")\n",
+    "  detached_actor = ray.get_actor(sys.argv[1])\n",
+    "  ray.kill(detached_actor)\n",
+    "  ```\n",
+    "\n",
+    "### Step 5: Trigger RayCluster scale-up by creating detached actors"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "f3e35aad-9afd-4f19-9b0c-62a80e3a30a0",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Defaulted container \"ray-head\" out of: ray-head, autoscaler\n",
+      "2025-04-10 09:22:39,435\tINFO worker.py:1514 -- Using address 127.0.0.1:6379 set in the environment variable RAY_ADDRESS\n",
+      "2025-04-10 09:22:39,436\tINFO worker.py:1654 -- Connecting to existing Ray cluster at address: 10.244.0.6:6379...\n",
+      "2025-04-10 09:22:39,446\tINFO worker.py:1832 -- Connected to Ray cluster. View the dashboard at \u001b[1m\u001b[32m10.244.0.6:8265 \u001b[39m\u001b[22m\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Step 5.1: Create a detached actor \"actor1\" which requires 1 CPU.\n",
+    "export HEAD_POD=$(kubectl get pods --selector=ray.io/node-type=head -o custom-columns=POD:metadata.name --no-headers)\n",
+    "kubectl exec -it $HEAD_POD -- python3 /home/ray/samples/detached_actor.py actor1"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "73d74d6b-865e-4d74-b095-84d42de0073e",
+   "metadata": {
+    "editable": true,
+    "slideshow": {
+     "slide_type": ""
+    },
+    "tags": [
+     "nbval-ignore-output",
+     "remove-cell"
+    ]
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "pod/raycluster-autoscaler-small-group-worker-h8tw4 condition met\n",
+      "raycluster-autoscaler-small-group-worker-h8tw4\n"
+     ]
+    }
+   ],
+   "source": [
+    "while true; do\n",
+    "    WORKER_POD1=$(kubectl get pods --selector=ray.io/node-type=worker -o custom-columns=POD:metadata.name --no-headers)\n",
+    "    if [[ -n \"$WORKER_POD1\" ]]; then\n",
+    "        export WORKER_POD1\n",
+    "        break\n",
+    "    fi      \n",
+    "    sleep 2\n",
+    "done\n",
+    "kubectl wait --for=condition=ready pod/$WORKER_POD1 --timeout=500s\n",
+    "echo $WORKER_POD1"
```

Comment:
I think we can use `kubectl wait` to wait for the `status.availableWorkerReplicas` field of `raycluster/raycluster-autoscaler` becomes to 1 instead of waiting for pod name becoming non-empty string.